### DISCUSSION TITLE
各位投票投票！！！！希望大佬出个关于python的nvim开发教程

### DISCUSSION THREAD

**Discussion by wxzvim** [nshen/learn-neovim-lua#78] [Polls]
各位投个票，让大佬，出个python的nvim教程吧

**Comment by nshen** [comment]
我也不会写python啊， 我觉得语言都差不多，就是装上 lsp基本就可以用了吧。
https://github.com/nshen/learn-neovim-lua/blob/v2/lua/lsp/config/pyright.lua

**Reply by wxzvim** [reply]
就是不清楚哪些工具是最好用的，网上搜索的一些nvim插件的python都说不维护呢

**Reply by allworldg** [reply]
@wxzvim  可以去reddit直接搜或者问,应该会比较多回答.(简单写python直接lsp完事了)

**Reply by Linya-IronMan** [reply]
我前端用来开发还有点小问题呢 ......
